Makefile

```diff
@@ -103,11 +103,11 @@ ifeq (, $(shell which controller-gen))
 	}
 CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
-CONTROLLER_GEN=$(shell which controller-gen)
+CONTROLLER_GEN=$(shell command -v controller-gen; )
```

Comment:
Very minor nitpick: all the other instances have a space then the ;  - this has a space after the semicolon.
The rest looks good.

Reply:
thanks for catching! updated